components/AddTranslationForm/AddTranslationForm.js

```diff
@@ -0,0 +1,94 @@
+import styled from "styled-components";
+import { InputField, Dropdown, BackButton, ActionButton } from "../StyledForm";
+import { languages } from "../../lib/languages";
+
+export default function AddTranslationForm({
+  onReturnFromEditMode,
+  onSaveTranslation,
+  id,
+}) {
+  function handleAddedTranslation(event) {
+    event.preventDefault();
+    const fields = event.target.elements;
+    const language2 = fields.queryLanguage2.value.split("-");
+    const translation2 = fields.queryLanguage2Translation.value;
+    const gender2 = fields.gender2.value;
+
+    const query2 = {
+      language: language2[1],
+      flag: language2[0],
+      translation: translation2,
+      gender: gender2,
+    };
+    onSaveTranslation(id, query2);
+    onReturnFromEditMode();
+  }
+
+  return (
+    <StyledForm onSubmit={handleAddedTranslation}>
+      <label htmlFor="queryLanguage1" name="queryLanguage1">
+        <Dropdown name="queryLanguage2" id="queryLanguage2" required>
```

Comment:
If you wrap your input inside a label, you don't need "htmlFor" and "id".  The reference is done via the nesting. And I am like 80% sure, the label-tag does not support the name-attribute.

Reply:
Thanks! I'll look into it!